src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -101,8 +118,135 @@ More information about these app properties, the types they
 depend on, and how internationalized Flutter apps are typically
 structured, can be found below.
 
+<a name="adding-localized-messages"></a>
+### Adding your own localized messages
+
+Once `flutter_localizations` has been introduced, a few extra
```

Comment:
- We do not currently have a "best practice" localized example anywhere -- I'll look into adding one somewhere.
- It doesn't have to be English, we can use Mandarin as the template, for example. It is simply a matter of what the developer would like to use. However, a template is necessary because the tool uses this as the basis for generating translations for the other locales that the developer intends to use. For example, the template could be in Mandarin, and the other supported locales could be Spanish and Malay.
-  The tool does not localize images. Do you have any resources for that that I could look into? To be quite honest, I'm not sure what about an image needs to be localized 😆

Reply:
I just added the small example and a link to it as part of this PR in the following commits:
Sample app: https://github.com/flutter/website/pull/4942/commits/e5b5f06aae7a2946c0a4b97dc58a5ad5ca215543
Link to sample app: https://github.com/flutter/website/pull/4942/commits/60c03dd8301f30349c32e1d46a3617e8328412c1